smallrye-reactive-messaging-vertx-eventbus/src/main/java/io/smallrye/reactive/messaging/eventbus/EventBusSink.java

```diff
@@ -20,13 +23,13 @@
     private final long timeout;
 
     EventBusSink(Vertx vertx, VertxEventBusConnectorOutgoingConfiguration config) {
-        this.vertx = Objects.requireNonNull(vertx, "Vert.x instance must not be `null`");
+        this.vertx = Objects.requireNonNull(vertx, msg.vertXInstanceMustNotBeNull());
```

Comment:
vertx, lowe-case x.

Reply:
thinking about it, this check is useless. The instance cannot be `null` since the inclusion of vert.x in the core.

Reply:
Should I just remove it ?

Reply:
Yes, same in the source. Remove the check, but with the assignment.

Reply:
Removed